### PR TITLE
[Elastic] Bump up `elastic` API version to `2023-02-01-preview`

### PR DESCRIPTION
**Pull request by Jing-song**
---
Bump up `elastic` API version to `2023-02-01-preview`
AAZ model https://github.com/Azure/aaz/pull/198
close https://github.com/Azure/azure-cli/issues/24618

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Elastic

File: src/elastic/azext_elastic/azext_metadata.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "azext.isExperimental": true,
-    "azext.minCliCoreVersion": "2.15.0"
+    "azext.minCliCoreVersion": "2.48.0"
```
Comment (kairu-ms) [inline src/elastic/azext_elastic/azext_metadata.json:3]:
```suggestion
    "azext.minCliCoreVersion": "2.49.0"
```

File: src/elastic/azext_elastic/aaz/latest/elastic/monitor/tag_rule/_create.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+
+@register_command(
+    "elastic monitor tag-rule create",
+)
+class Create(AAZCommand):
+    """Create a tag rule set for a given monitor resource.
+
+    :example: Create monitor tag-rule
+        az elastic monitor tag-rule create -n default -g rg --monitor-name monitor --log-rules {filteringTags:[{name:Environment,value:Prod,action:Include}]}
```
Comment (kairu-ms) [inline src/elastic/azext_elastic/aaz/latest/elastic/monitor/tag_rule/_create.py:21]:
```suggestion
        az elastic monitor tag-rule create -n default -g rg --monitor-name monitor --log-rules "{filteringTags:[{name:Environment,value:Prod,action:Include}]}"
```

**Review by kairu-ms — APPROVED** [review]

File: src/elastic/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.0
```
Comment (kairu-ms) [inline src/elastic/HISTORY.rst:6]:
```suggestion
1.0.0b1
```

File: src/elastic/setup.py [hunk 4]
```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.1.0'
+VERSION = '1.0.0'
```
Comment (kairu-ms) [inline src/elastic/setup.py:13]:
```suggestion
VERSION = '1.0.0b1'
```

**Review by kairu-ms — CHANGES_REQUESTED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**